include/gsl/gsl_byte

```diff
@@ -33,6 +33,8 @@
 
 #include <type_traits>
 
+// VS2017 15.8 added support for the __cpp_lib_byte definition
+// TODO: drop _HAS_STD_BYTE when support for pre 15.8 expires
```

Comment:
Note that the IDE now has a Task List that detects TODO comments; you may not want to use TODO in your headers.

Reply:
Thanks, I'll clean that up

Reply:
Conversely, you may want to use `TODO` even *more* in your headers in hopes of tricking users into submitting patches.

Reply:
Updated